props/saber_fett263_buttons.h

```diff
@@ -3924,7 +3918,13 @@ SaberFett263Buttons() : PropBase() {}
     battle_mode_ = true;
 #endif
   }
-	
+
```

Comment:
This looks like it came from the other PR?

Reply:
ah, yup, I'll fix